Automattic/wp-calypso
# wp-admin Command Palette: Add "Send feedback" command

Fixes https://github.com/Automattic/wp-calypso/issues/87940

## Proposed Changes

Adds the "Send feedback" command to the wp-admin command palette.

<img width="438" alt="Screenshot 2024-02-28 at 17 37 23" src="https://github.com/Automattic/wp-calypso/assets/1233880/3cca32cb-f92a-48ef-ae1b-622910dc8a9d">


## Testing Instructions

- Apply the latest version of https://github.com/Automattic/jetpack/pull/35635 to your test site
- Run `install-plugin.sh command-palette add/command-palette-wp-admin-send-feedback` in your sandbox
- Sandbox `widgets.wp.com`
- Visit the WP Admin of your test site
- Open the command palette
- Search the "Send feedback" command
- Make sure it shows up
- Execute it
- Make sure it opens the contact form of the help center
